docs/operations-guide/start.md

```diff
@@ -280,15 +280,17 @@ By default, Metabase will be listening on `localhost`.  In some production envir
 
 ### Using HTTPS with Metabase
 
-If you have an ssl certificate and would prefer to have Metabase run over HTTPS directly using its webserver you can do so by using the following environment variables:
+If you have an SSL certificate and would prefer to have Metabase run over HTTPS directly using its webserver you can do so by using the following environment variables:
 
     export MB_JETTY_SSL="true"
-    export MB_JETTY_SSL_Port="8443"
-    export MB_JETTY_SSL_Keystore="path/to/keystore.jks"
-    export MB_JETTY_SSL_Keystore_Password="storepass"
+    export MB_JETTY_SSL_PORT="8443"
+    export MB_JETTY_SSL_KEYSTORE="path/to/keystore.jks" # replace these values with your own
```

Comment:
Maybe `# replace with the path to your Java KeyStore`

And if you're gonna do line comments, might as well do one for the password line below, like `# replace with the password for your Java KeyStore`

(Another way to go would be to forego line comments entirely and just rely on the explanatory paragraph below.)